Userland/Libraries/LibWeb/Infra/Strings.cpp

```diff
@@ -29,4 +31,38 @@ String strip_and_collapse_whitespace(StringView string)
     return builder.string_view().trim(Infra::ASCII_WHITESPACE);
 }
 
+// https://infra.spec.whatwg.org/#code-unit-prefix
+bool is_code_unit_prefix(String const& potential_prefix, String const& input)
```

Comment:
Ah, I thought I commented this earlier but it got lost - these can be StringViews, we don't need allocated strings